### DISCUSSION TITLE
如何只在一天的某些时间段进行下载？

### DISCUSSION THREAD

**Discussion by breathiness** [yutto-dev/yutto#63] [Q&A] [answered]
我打算把我的收藏夹全部下载下来，估计会需要很久。这个应该会影响到其他设备的上网速度。  
请问要怎么样才能让下载任务只在特定时间段进行下载呢？

**Comment by SigureMo** [comment, marked as answer]
这我觉得就完全超出 yutto 本身的职责了，yutto 再做个计划任务系统感觉就不太合适了……

这个完全可以由一个上层的任务调度程序来控制，由上层程序来控制 yutto 的启停。其实它只需要在到达时间启动 yutto，超出时间范围时 kill 掉 yutto 就好，因为 yutto 默认的断点续传可以直接接着上次停止的位置继续下载（如果你没有对下载目录进行任何改动的话），所以你不必担心如果重新下载的话之前的都白下载了的问题。

**Reply by breathiness** [reply]
原来如此，可以直接续传的话确实可以。我写个小脚本试试。
